util/libcrypto.num

```diff
@@ -4379,3 +4379,7 @@ BN_priv_rand_range                      4322	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_normalize                     4323	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_cmp_time_t                    4324	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_compare                       4325	1_1_1	EXIST::FUNCTION:
+ossl_ctype_check                        4326	1_1_1	EXIST::FUNCTION:
+ossl_tolower                            4327	1_1_1	EXIST::FUNCTION:
+ossl_toupper                            4328	1_1_1	EXIST::FUNCTION:
+ossl_isalnum                            4329	1_1_1	EXIST::FUNCTION:
```

Comment:
libssl requires `isalnum`.  That's the primary reason for the export.  This is one instance where an extra class might be useful.

I'll revert the changes in apps.